prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm.c

```diff
@@ -413,16 +410,18 @@ fi_ibv_rdm_tagged_process_recv(struct fi_ibv_rdm_ep *ep,
 					 struct fi_ibv_rdm_tagged_request,
 					 queue_entry);
 
+			const int data_len = arrived_len -
+				sizeof(struct fi_ibv_rdm_tagged_header);
+
 			if (found_request->len < data_len) {
-				FI_IBV_ERROR
-				    ("%s: %d RECV TRUNCATE, data_len=%d, posted_len=%d, "
-				     "conn %p, tag 0x%llx, tagmask %llx\n",
-				     __FUNCTION__, __LINE__, data_len,
-				     found_request->len, found_request->conn,
-				     (long long unsigned int)found_request->tag,
-				     (long long unsigned int)found_request->
-				     tagmask);
-				abort();
+				VERBS_INFO(FI_LOG_EP_DATA,
+					"%s: %d RECV TRUNCATE, data_len=%d, posted_len=%d, "
+					"conn %p, tag 0x%llx, tagmask %llx\n",
+					__FUNCTION__, __LINE__, data_len,
+					found_request->len, found_request->conn,
+					found_request->tag,
+					found_request->tagmask);
+//				return -FI_EOTHER;
```

Comment:
Note that the return statement is commented out.  I'm not sure if this is intentional.  Would an assert() be useful here?